tools/licensescan/main.go

```diff
@@ -125,7 +129,14 @@ func RunLicenseScan(ctx context.Context, opts RunLicenseScanOptions) error {
 		modules = append(modules, module)
 
 		p := filepath.Join("modules", module.Name, module.Version+".yaml")
-		b, err := ioutil.ReadFile(p)
+		b, err := os.ReadFile(p)
+		if err != nil && os.IsNotExist(err) {
+			b2, err2 := modulesFS.ReadFile(p)
+			if err2 == nil {
+				b = b2
```

Comment:
so this will ensure license info from the moduleFS gets written back to local FS.

Reply:
Yes, ish.  The workflow I use is to run it against the binary.  Then I search for `license: TODO` go through and fill in any missing licenses that are generated.  The link to the go package DB makes this easy, if a little tedious.  Then I commit the files, and then yes the `go:embed` ensures that (once merged) they are automatically embedded in as part of go build.